cyclonedx/model/__init__.py

```diff
@@ -217,4 +326,777 @@ def get_url(self) -> str:
         return self._url
 
     def __repr__(self) -> str:
-        return f'<ExternalReference {self._reference_type.name}, {self._url}> {self._hashes}'
+        return f'<ExternalReference {self._type.name}, {self._url}> {self._hashes}'
+
+
+class IssueClassification(Enum):
```

Comment:
suggestion: put the Issue-related models in an own module(file)?

Reply:
Great suggestion - done and PR updated.